extensions/amp-analytics/amp-analytics.md

```diff
@@ -59,6 +59,13 @@ when the document is first loaded, and each time an `<a>` tag is clicked:
 </amp-analytics>
 ```
 
+#### <a name="attributes"></a>Attributes
+
```

Comment:
Sure. You OK with me creating an issue for second iteration? I'll assign it to myself and cc you.